proto/covidshield.proto

```diff
@@ -15,10 +15,10 @@ option go_package="pkg/proto/covidshield";
 // will fail.
 message KeyClaimRequest {
   // one_time_code is the code received from the testing portal.
-  required string one_time_code = 1; // 8 numerical digits
+  optional string one_time_code = 1; // 8 numerical digits
```

Comment:
What does making these optional do?
Is it connected to the Ruby on Rails Protoc stuff or separate? If separate perhaps we should split it up in a different PR?

Reply:
Ahh I see, it's unrelated but still useful. Sounds good :)